README.md

```diff
@@ -147,7 +147,7 @@ Here are the brief instructions for CUDA/CuDNN, for details and other options se
 Then either:
 
 7. Hit `Enter` to build it.
-8. Resulting binary will be `build/lc0.exe`
+8. Resulting binary will be in `build/lc0.exe`
```

Comment:
This correction is wrong, as it implies `build/lc0.exe` would be a directory and not a binary.